js/src/components/paid-ads/ads-campaign.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import StepContent from '.~/components/stepper/step-content';
+import StepContentHeader from '.~/components/stepper/step-content-header';
+import StepContentFooter from '.~/components/stepper/step-content-footer';
+import AppDocumentationLink from '.~/components/app-documentation-link';
+import AppButton from '.~/components/app-button';
+import { useAdaptiveFormContext } from '.~/components/adaptive-form';
+import CreateCampaignFormContent from '.~/components/paid-ads/create-campaign-form-content';
+
+/**
+ * Renders the form container for campaign management.
+ *
+ * Please note that this component relies on an AdaptiveForm's context, so it expects
+ * a context provider component (`AdaptiveForm`) to existing in its parents.
+ *
+ * @fires gla_documentation_link_click with `{ context: 'create-ads', link_id: 'see-what-ads-look-like', href: 'https://support.google.com/google-ads/answer/6275294' }`
+ *
+ * @param {Object} props React props.
+ * @param {() => void} props.onContinue Callback called once continue button is clicked.
+ */
+export default function AdsCampaign( { onContinue } ) {
+	const formContext = useAdaptiveFormContext();
+	const { isValidForm } = formContext;
+
+	return (
+		<StepContent>
+			<StepContentHeader
+				title={ __(
+					'Create your paid campaign',
+					'google-listings-and-ads'
+				) }
+				description={ createInterpolateElement(
+					__(
+						'Paid Performance Max campaigns are automatically optimized for you by Google. <link>See what your ads will look like.</link>',
+						'google-listings-and-ads'
+					),
+					{
+						link: (
+							<AppDocumentationLink
+								context="create-ads"
+								linkId="see-what-ads-look-like"
+								href="https://support.google.com/google-ads/answer/6275294"
+							/>
+						),
+					}
+				) }
+			/>
+			<CreateCampaignFormContent formProps={ formContext } />
```

Comment:
Since we changed the names from `CreatePaidAdsCampaignForm` to `AdsCampaign`. WDYT of changing this to `AdsCampaignForm` or `AdsCampaignFormContent`?

Reply:
Same for the SCSS

Reply:
P.S. Reverting the previous name to `CreateAdsCampaign` also ok for me.

Reply:
Actually, thinking about it, I think it would be better `CreateAdsCampaign`. So then we would have `CreatePaidAdsCampaign`, `EditPaidAdsCampaignFormContent` and `CreatePaidAdsCampaignFormContent`. In any case, I miss some consistency here.

Reply:
This component will be also used to replace the `EditPaidAdsCampaignForm` in [edit-paid-ads-campaign/index.js](https://github.com/woocommerce/google-listings-and-ads/blob/7ccbc8f8e5568a6f74bbd15d337fbae286ca05f6/js/src/pages/edit-paid-ads-campaign/index.js#L69) in a subsequent PR. I initially named it `ManageAdsCampaign`, later, considering that most of the component names in [paid-ads](https://github.com/woocommerce/google-listings-and-ads/tree/7ccbc8f8e5568a6f74bbd15d337fbae286ca05f6/js/src/components/paid-ads) don't start with a verb, consistency considerations here led to the name `AdsCampaign`.

In addition, there is no clear consistency in the naming of this aspect in GLA codebase, such as `AttributeMappingRuleModal` doesn't start with a verb either.

And the reason why there is no component name with "FormContent" is described [here](https://github.com/woocommerce/google-listings-and-ads/pull/1794#discussion_r1035576050). Borrowing from the same example again, I doubt a name like `AttributeMappingFormRuleModal` or `AttributeMappingRuleFormContentModal`, would be more appropriate.

Reply:
Let's see then when it's all finish. Hard to discuss without all the components in the folder and knowing what each component finally contains.

Not sure neither whats the relationship with `AttributeMappingRuleModal` here.